frontend/src/pages/pipelines/global/modelCustomization/trainingHardwareSection/TrainingHardwareProfileFormSection.tsx

```diff
@@ -81,20 +79,22 @@ const TrainingHardwareProfileFormSection: React.FC<TrainingHardwareProfileFormSe
                 <>
                   <Content component={ContentVariants.p}>
                     This list includes only hardware profiles with defined GPUs.
-                    {isProjectScoped &&
-                      projectScopedHardwareProfiles['1'] &&
-                      projectScopedHardwareProfiles['0'].length > 0 && (
-                        <List>
-                          <ListItem>
+                  </Content>
+                  {isProjectScoped &&
+                    projectScopedHardwareProfiles['1'] &&
+                    projectScopedHardwareProfiles['0'].length > 0 && (
+                      <Content>
```

Comment:
Use `<Content component={ContentVariants.ul}>` and `<Content component={ContentVariants.li}>` to replace `ul` and `li` here, avoid using the native HTML element.